code/WorkInProgress/Mini/ATM.dm

```diff
@@ -340,7 +341,7 @@ log transactions
 						usr << "<span class='notice'>The [src.name] flashes an error on its display.</span>"
 						return
 					lastprint = world.timeofday
-					var/obj/item/weapon/paper/R = new(src.loc)
+					var/obj/item/weapon/paper/R = new /obj/item/weapon/paper(src.loc)
```

Comment:
You don't need to change it.
All the code in this section uses usr, and it's a Topic, so it's pretty much standard.